README.md

```diff
@@ -76,6 +76,11 @@ Helix can be installed on MacOS through homebrew:
 brew install helix
 ```
 
+## Minimum Stable Rust Version (MSRV) Policy
+
+Helix follows the MSRV of firefox.
+The current MSRV and future changes to the MSRV are listed in the [firefox documentation](https://firefox-source-docs.mozilla.org/writing-rust-code/update-policy.html).
```

Comment:
```suggestion
Helix follows the MSRV of Firefox.
The current MSRV and future changes to the MSRV are listed in the [Firefox documentation](https://firefox-source-docs.mozilla.org/writing-rust-code/update-policy.html).
```

Reply:
I'm not sure this needs such an explicit mention in the README. I would either put it as a comment in `rust-toolchain.toml` or in `CONTRIBUTING.md` since it's an implementation detail.

Reply:
I think having the MSRV policy document in a visisble place is helpful (both for package maintainers aswell as developers) but I agree that `CONTRIBUTING.md ` is probably the better place. I have moved the the section on MSRV from `README.md` to `CONTRIBUTING.md~ (and fixed the spelling)